File: yumyum-back/src/main/java/com/yumyum/domain/user/dto/UserResponse.java
package com.yumyum.domain.user.dto;

import com.yumyum.domain.user.entity.User;
import lombok.*;

@Data
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class UserResponse {
    private Long id;
    private String email;
    private String nickname;
    private String introduction;
    private String profilePath;

    public UserResponse(final User user){
        this.id = user.getId();
        this.email = user.getEmail();
        this.nickname = user.getNickname();
        this.introduction = user.getIntroduction();
        this.profilePath = user.getProfilePath();
    }
}

File: AOS/app/src/main/java/com/omnyom/yumyum/model/userInfo/UserResponse.kt
package com.omnyom.yumyum.model.userInfo

data class UserResponse(
    val `data`: UserData,
    val message: String,
    val status: String
)

data class UserData(
        val email: String,
        val id: Int,
        val introduction: String,
        val nickname: String,
        val profilePath: String
)